src/main.cpp

```diff
@@ -1,6 +1,6 @@
 #include <vector>
 #include <iostream>
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include "functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
```

Comment:
You don't need this comment.

Reply:
move header up

Reply:
Sort headers that way:
```#include <iostream>
#include <vector>
#include "functions.hpp"```
It's good according to Mateusz:

https://coders.school/edu/courses/powtorka-1-qa/lesson/kolejnosc-include/

Other example:

https://github.com/coders-school/kurs_cpp_podstawowy/blob/module2/module1/homework/fibonacci/main.cpp

Reply:
@Marcinati, thanks for explaining the include order, it should be fixed now.